### PR TITLE
fix(web.domain): correct the display of domain contact info

### PR DESCRIPTION
**Pull request by vmesnyovh**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #MANAGER-13724
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

This PR addresses the incorrect display of domain contact information, where contact details were shown as JSON objects instead of plain IDs. 
Changes ensure compatibility with both data structures (object with id and direct string formats) across Administrator, Technical, and Billing contact sections.

### CODE REVIEW

File: packages/manager/apps/web/client/app/domain/general-informations/GENERAL_INFORMATIONS.html [hunk 1]
```diff
@@ -480,7 +480,23 @@
                 >
                     <oui-tile-description>
                         <ul class="list-unstyled">
-                            <p class="mb-0">
+                            <p
+                                class="mb-0"
+                                ng-if="::$ctrl.domainInfos.contactAdmin.id"
```
Comment (sachinrameshn) [inline packages/manager/apps/web/client/app/domain/general-informations/GENERAL_INFORMATIONS.html:485]:
```suggestion
                                data-ng-if="::$ctrl.domainInfos.contactAdmin.id"
```

File: packages/manager/apps/web/client/app/domain/general-informations/GENERAL_INFORMATIONS.html [hunk 2]
```diff
@@ -480,7 +480,23 @@
...
+                                ></span
+                                >:
+                                <span
+                                    class="small"
+                                    data-translate="global_admin"
+                                ></span>
+                            </p>
+                            <p
+                                class="mb-0"
+                                ng-if="!$ctrl.domainInfos.contactAdmin.id"
```
Comment (sachinrameshn) [inline packages/manager/apps/web/client/app/domain/general-informations/GENERAL_INFORMATIONS.html:498]:
```suggestion
                                data-ng-if="!$ctrl.domainInfos.contactAdmin.id"
```

File: packages/manager/apps/web/client/app/domain/general-informations/GENERAL_INFORMATIONS.html [hunk 3]
```diff
@@ -490,7 +506,24 @@
                                     data-translate="global_admin"
                                 ></span>
                             </p>
-                            <p class="mb-0">
+
+                            <p
+                                class="mb-0"
+                                ng-if="::$ctrl.domainInfos.contactTech.id"
```
Comment (sachinrameshn) [inline packages/manager/apps/web/client/app/domain/general-informations/GENERAL_INFORMATIONS.html:512]:
```suggestion
                                data-ng-if="::$ctrl.domainInfos.contactTech.id"
```

**Review by anooparveti — CHANGES_REQUESTED** [review]
**question:** @vmesnyovh do we have approval for the change?

**Comment by vmesnyovh** [comment]
> **question:** @vmesnyovh do we have approval for the change?

Hello @anooparveti the change was requested by Antoine Calloch.
The problem is that this affects some of the prod users, and more and more people are affected as the deployment progresses.

**Review by anooparveti — APPROVED** [review]

**Review by sachinrameshn — APPROVED** [review]

**Pull request merged**